test/end2end/pub_server_data.dart

```diff
@@ -304,7 +304,17 @@ final _data = {
     'oldAnalysisOptions': false,
     'isExperimentalVersion': false,
     'isPreReleaseVersion': true,
-    'toolIssueCount': 0,
+    'errorCount': 0,
+    'warningCount': 1,
+    'hintCount': 0,
   },
   'fitness': {'magnitude': 455.0, 'shortcoming': 1.0},
+  'suggestions': [
+    {
+      'level': 'warning',
+      'title': 'Use `analysis_options.yaml`.',
+      'description':
+          'Static analysis will help you to detect bugs and potential issues earlier.',
```

Comment:
We're analyzing (in pana) all the time. The important bit is that the user has strong-mode enabled.

That really should be the check here...

Reply:
PTAL, added strong-mode check.